cmd/crates/soroban-spec-typescript/src/types.rs

```diff
@@ -250,6 +252,7 @@ impl From<&ScSpecEntry> for Entry {
                 name: e.name.to_utf8_string_lossy(),
                 cases: e.cases.iter().map(Into::into).collect(),
             },
+            ScSpecEntry::EventV0(_) => todo!("EventV0 is not implemented yet"),
```

Comment:
Leaving the todo in here will make the CLI panic if it encounters a spec that contains an event. Ideally we fill this in, but if we can't yet, I think we need to do something else, such as eprintln! a warning that the spec generated is incomplete, or silently hide the ommission. Otherwise it won't be possible to call anything that uses this code with newer contracts.

If we're releasing only an rc, then this is probably fine temporarily.

Reply:
I'm going to keep it like this for this initial release, and I'll work on that tomorrow. :)